packages/upload/test/adding-files.common.js

```diff
@@ -102,6 +102,18 @@ describe('adding files', () => {
       expect(upload._dragoverValid).to.be.false;
     });
 
+    it('should not have dragover property when disabled', async () => {
```

Comment:
```suggestion
    it('should not have dragoverValid property when disabled', async () => {
```

Reply:
Updated the names of this test and a related one. The test setup here is not great with testing some internal properties when attributes or event details could be used. But that results in several changes, I think I'll open a follow-up PR to clean those up.